changelog/unreleased/kong/refactor_dns_client.yml

```diff
@@ -2,8 +2,6 @@ message: >
   Starting from this version, a new DNS client library has been implemented and added into Kong. The new DNS client library has the following changes
   - Introduced global caching for DNS records across workers, significantly reducing the query load on DNS servers.
   - Introduced observable statistics for the new DNS client, and a new Admin API `/status/dns` to retrieve them.
-  - Deprecated the `dns_no_sync` option. Multiple DNS queries for the same name will always be synchronized (even across workers). This remains functional with the legacy DNS client library.
-  - Deprecated the `dns_not_found_ttl` option. It uses the `dns_error_ttl` option for all error responses. This option remains functional with the legacy DNS client library.
-  - Deprecated the `dns_order` option. By default, SRV, A, and AAAA are supported. Only names in the SRV format (`_service._proto.name`) enable resolving of DNS SRV records.
+  - Introduced new directives `resolver_*`, see the detailed documentation in `kong.conf.default`.
```

Comment:
Not sure whether the end-user has the `kong.conf.default` file.

Reply:
It seems that new feature does not need to describe the new directives in its changelog, for example, https://github.com/Kong/kong-ee/pull/7599/files

Reply:
remove line associated to directives